rules/config

```diff
@@ -131,6 +131,9 @@ ENABLE_RESTAPI = n
 # ENABLE_NAT - build docker-sonic-nat for nat support
 ENABLE_NAT = y
 
+# SONIC_TELEMETRY_WRITABLE - Enable write/config operations via the gNMI interface.
+# Uncomment to enable:
+# SONIC_TELEMETRY_WRITABLE = y
```

Comment:
Keeping in line with the other options, I suggest removing the `SONIC_` prefix from the user-facing option name.

Reply:
I previously had it without SONIC_ prefix and was told to add the SONIC_ prefix

Reply:
Sorry. It seems like there must have been some confusion. For the user-facing option name, it seems redundant and a waste of screen space to prefix them all with "SONIC_". It appears as though some people haven't been consistent with this, however. We'll need to go through and clean some up in the future.

Reply:
Joe would it be OK to drop SONIC_ prefix from both internal & user-facing  ? 
This would make it cleaner and less error-prone.

Reply:
Yes. I don't see a problem with removing it from both, unless other internal processing would need to be done before setting the internal variable, in which case, different names would make sense.

Reply:
@seiferteric , can we get this resolved?

Reply:
Okay done, also updated the corresponding telemetry PR https://github.com/Azure/sonic-telemetry/pull/34